### PR TITLE
Fixes #226 - Deprecate Bower

### LINKED ISSUE #226: Deprecate Bower
Can you deprecate bower and move to yarn?

### PR DESCRIPTION
**Pull request by willsteinmetz**
Bower is only being maintained now and not further developed according to their website: https://bower.io/ They recommend Yarn and Webpack. However, NPM continually keeps adding what made Yarn a plus to the main project so I don't think that makes sense to add it as part of this PR if it's deemed worth adding as a flag at generation time.

### CODE REVIEW

**Comment by willsteinmetz** [comment]
I don't know why the Travis tests failed. The exact same test command ran just fine locally.

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +14,6 @@ cache:
     - $(npm config get prefix)/bin/yo
     - temp/one/node_modules
     - temp/two/node_modules
-before_script:
-  - npm install -g yo bower
```
Comment (petecoop) [inline .travis.yml]: Test fails because you removed the installation of `yo`

**Comment by petecoop** [comment]
I support this, all fresh projects I've made have not used bower and have been migrating older projects away from it and to use npm and other tools e.g. gulp/webpack to copy over files if needed from node_modules

**Comment by willsteinmetz** [comment]
Uh...failed on Node 4 and 5 but ran on the others. That's odd...

**Comment by petecoop** [comment]
```
/home/travis/.nvm/versions/node/v4.9.1/lib/node_modules/yo/node_modules/yeoman-environment/lib/resolver.js:158
    let lookup = path.join(...parts.slice(0, i + 1), 'node_modules');
```
Looks like yo no longer supports those versions of node! Just because they are using `...`

**Comment by willsteinmetz** [comment]
I mean, that kind of makes sense since versions 4 and 5 are old. Version 10 is the latest and 8 is the LTS. Does it make sense to lop off 4 and 5 from the Travis runner in this PR and possibly add 9 and 10?

**Comment by petecoop** [comment]
yep looks good, no point in supporting old

**Comment by petecoop** [comment]
I've released this and other recent updates as 2.17.0!

**Pull request merged**